msfvenom

```diff
@@ -270,20 +270,20 @@ if __FILE__ == $0
     generator_opts[:list].each do |mod|
       case mod.downcase
         when "payloads"
-          $stderr.puts dump_payloads
+          $stdout.puts dump_payloads
         when "encoders"
-          $stderr.puts dump_encoders(generator_opts[:arch])
+          $stdout.puts dump_encoders(generator_opts[:arch])
         when "nops"
-          $stderr.puts dump_nops
+          $stdout.puts dump_nops
         when "all"
           # Init here so #dump_payloads doesn't create a framework with
           # only payloads, etc.
           init_framework
-          $stderr.puts dump_payloads
-          $stderr.puts dump_encoders
-          $stderr.puts dump_nops
+          $stdout.puts dump_payloads
+          $stdout.puts dump_encoders
+          $stdout.puts dump_nops
         else
-          $stderr.puts "Invalid module type"
+          $stdout.puts "Invalid module type"
```

Comment:
I fat-fingered that one. Yeah it should be stderr. Could you please change that when you merge?